Source/Core/DolphinNoGUI/PlatformX11.cpp

```diff
@@ -166,11 +169,9 @@ void PlatformX11::UpdateWindowPosition()
 
   Window winDummy;
   unsigned int borderDummy, depthDummy;
-  XGetGeometry(m_display, m_window, &winDummy, &SConfig::GetInstance().iRenderWindowXPos,
-               &SConfig::GetInstance().iRenderWindowYPos,
-               reinterpret_cast<unsigned int*>(&SConfig::GetInstance().iRenderWindowWidth),
-               reinterpret_cast<unsigned int*>(&SConfig::GetInstance().iRenderWindowHeight),
-               &borderDummy, &depthDummy);
+  XGetGeometry(m_display, m_window, &winDummy, &m_window_x, &m_window_y,
+               reinterpret_cast<unsigned int*>(&m_window_width),
+               reinterpret_cast<unsigned int*>(&m_window_height), &borderDummy, &depthDummy);
```

Comment:
Declaring `m_window_width/height` as unsigned int would eliminate this ugly reinterpret_cast.